utils/manager.py

```diff
@@ -143,11 +158,16 @@ async def AIResponse(self, userInput):
 
         reply, character = await self.removeKeywords(response)
 
-        # Log AI input
-        self.chathistory.append({"role": "assistant", "content": response})
+        if reply == "ERROR":
+            response = await self.getDummyResponse()
+            reply, character = await self.removeKeywords(response)
```

Comment:
You could have "reply, character = await self.removeKeywords(response)" at the end of the method instead of having it repeat